python/ray/dashboard/modules/job/utils.py

```diff
@@ -53,7 +53,7 @@ def strip_keys_with_value_none(d: Dict[str, Any]) -> Dict[str, Any]:
 
 def redact_url_password(url: str) -> str:
     """Redact any passwords in a URL."""
-    secret = re.findall("https?:\/\/.*:(.*)@.*", url)
+    secret = re.findall("https?:\\/\\/.*:(.*)@.*", url)
```

Comment:
we can reuse `r""` so we don't need to escape

Reply:
sure. using raw string.